scratch3/blocks.js

```diff
@@ -705,10 +705,10 @@ ScriptView.prototype.draw = function(inside) {
   }
   this.height = y + 1
   if (!inside && !this.isFinal) {
-    this.height += block.hasPuzzle ? 44 : 36
+    this.height += block.hasPuzzle ? 8 : 0
   }
   if (!inside && block.isGlow) {
-    this.height += 2 // TODO unbreak this
+    this.height += 7 // TODO unbreak this
```

Comment:
This change is required when using `+` diff.

NOTE: this wasn't problematic because each script has redundant space due to `this.height += block.hasPuzzle ? 44 : 36`.